jekyll/_cci2/get-started-with-the-vs-code-extension.adoc

```diff
@@ -48,12 +48,12 @@ image::{{site.baseurl}}/assets/img/docs/vs_code_extension_login.png[Log in to Ci
 [#set-up-your-project-in-vs-code]
 == 3. Set up your project in VS Code
 
-NOTE: **Using GitLab?** The pipelines manager does not yet support GitLab projects. You can still use the xref:#open-a-config-file-to-test-language-support-features[config helper] for config writing and troubleshooting.
-
 If your VS Code workspace contains one or more CircleCI projects, the extension will detect them automatically, and the pipelines panel will be populated with your most recent pipelines.
 
 If no project is detected, open the extension's settings page (either through the VS Code command `CircleCI: Open Settings`, or by clicking on the settings icon (icon:cog[]) at the top of the pipelines panel), and select your projects manually.
 
+NOTE: **Using GitLab?** The pipelines manager does not yet support automatic detection for GitLab projects. Please select your project manually from the settings page.
```

Comment:
```suggestion
NOTE: **GitHub App and GitLab projects:** The pipelines manager does not yet support automatic detection for xref:github-apps-integration#[GitHub App] and xref:gitlab-integration#[GitLab] projects. Please select your project manually from the settings page.
```
In the changes we've made to other pages we refer to GitHub App and GitLab integrations, projects and/or users, and then we link to the integration pages which should help GitHub users determine whether they are on GH OAuth or GH App. Something like this should work... I think it's a bit less ambiguous compared to saying "Using GitHub App or GitLab?"